Library/Formula/postgresql.rb

```diff
@@ -72,13 +74,14 @@ def install
       ENV.append %w{CFLAGS LDFLAGS}, "-arch #{Hardware::CPU.arch_32_bit}"
     end
 
+    system "pkill postgres || true"
     system "./configure", *args
     system "make", "install-world"
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    unless File.exist? "#{var}/postgres-#{version}"
+      system "#{bin}/initdb", "#{var}/postgres-#{version}"
     end
```

Comment:
How does `postgres` know to look in this folder? We'll need to migrate existing data for 9.4 users?